realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -55,7 +55,7 @@
             "io.realm.internal.OsSchemaInfo",
             "io.realm.internal.OsObjectSchemaInfo",
             "io.realm.internal.Property",
-            "io.realm.internal.ProxyUtils",
+            "io.realm.ProxyUtils",
```

Comment:
:o

Reply:
It is package private, so the class doesn't leak into the public API